webapp/hooks/useStakeTokens.ts

```diff
@@ -4,11 +4,32 @@ import { isStakeToken } from 'utils/token'
 
 import { useHemi } from './useHemi'
 
+// TODO: It's pending to implement some missing badges
+// Issue #799 https://github.com/hemilabs/ui-monorepo/issues/799
+const tokenRewardMap: Record<string, string[]> = {
```

Comment:
here you can even do 
```suggestion
const tokenRewardMap: Record<StakeProtocols, StakeToken['extensions']['rewards']> = {
```

being `StakeProtocols` imported from `types/stake`

Reply:
you're correct, Protocols was incorrect 😅  Thanks!